tests/unit/adapters/application-test.js

```diff
@@ -940,7 +940,7 @@ test('#fetch handles 200 (Success) response status', function(assert) {
   const adapter = this.subject({type: 'posts', url: '/posts'});
   mockFetchJax(sandbox, adapter, postMock);
   sandbox.stub(adapter, 'cacheResource', function () {});
-  adapter.serializer = { deserialize: sandbox.spy(), deserializeIncluded: Ember.K };
+  adapter.serializer = { deserialize: sandbox.spy(), deserializeIncluded() {} };
```

Comment:
@cibernox nice, thanks for the fix. So named function declaration in shorthand syntax . Sweet. `deserializeIncluded() {}` and clean